packages/common/src/authentication/api/api.ts

```diff
@@ -74,13 +74,15 @@ export const getAuthQueries = (sdk: Sdk) => ({
       return refreshTokenGuard(result);
     },
     me: async (token?: string) => {
-      const result = await sdk.me(
-        {},
-        {
-          Authorization: `Bearer ${token}`,
-        }
-      );
-      return result.me;
+      try {
+        const result = await sdk.me(
+          {},
+          {
+            Authorization: `Bearer ${token}`,
+          }
+        );
+        return result.me;
+      } catch {}
```

Comment:
the login page was returning the 'server connection' error.. and then this query was throwing another error which hid the message. we don't need this one to toast, as auth issues are handled elsewhere